lib/cfa/base_model.rb

```diff
@@ -51,7 +51,7 @@ def save(changes_only: false)
     # @raise a *parser* specific error. If the parsed String is malformed, then
     #   depending on the used parser it may raise an error.
     def load
-      self.data = @parser.parse(@file_handler.read(@file_path))
+      self.data = @parser.parse(@file_handler.read(@file_path), @file_path)
```

Comment:
Ugh, this breaks with config_files_api_grub2, whose parser expects a single argument :-/

Either we simply change the other gem too, or we allow both APIs and here in the caller dynamically check the #parse method arity and only add the file path if supported...